meerk40t/gui/wxmeerk40t.py

```diff
@@ -327,9 +327,9 @@ def gui_start(**kwargs):
 supported_languages = (
     ("en", u"English", wx.LANGUAGE_ENGLISH),
     ("it", u"italiano", wx.LANGUAGE_ITALIAN),
-    ("fr", u"fran�ais", wx.LANGUAGE_FRENCH),
+    ("fr", u"français", wx.LANGUAGE_FRENCH),
     ("de", u"Deutsch", wx.LANGUAGE_GERMAN),
-    ("es", u"espa�ol", wx.LANGUAGE_SPANISH),
+    ("es", u"español", wx.LANGUAGE_SPANISH),
     ("zh", u"Chinese", wx.LANGUAGE_CHINESE),
```

Comment:
What changed here? The non utf-8 characters have a nasty habit of changing to a different broken set of characters so these changes concern me like it's trying to use unicode for these in a breaking fashion.

Reply:
Sorry - I didn't change these lines myself. I think Github changed it to UTF when I edited it in a browser - and I don't know any way of stopping this.

I am not unclear why we are using ISO 8859-1 rather than UTF anyway.

Reply:
I will leave you to correct this and merge it.

Reply:
It is on my ToDo list to stop using Github web pages and point my Git at my run-from-source directory. I just haven't got around it it yet.

Reply:
I'm fine with anything that actually shows the letters rather than the messed up thing they turn into. I'll likely be checking this prior to new versions though. Also hit this problem when I tried to use a tau with regard to angles in the rotate.